### PR TITLE
Pwx 4328 env vars

### PR DESCRIPTION
**Pull request by jrivera-px**
Hey so rather than add a separate page it was suggested that I add the environments to the options page. So I did that and we we can add link to the location for environment variables. Hope this is ok.

I am not sure how to add this to DCOS.

### CODE REVIEW

File: _includes/cmdargs.md [hunk 1]
```diff
@@ -1,34 +1,73 @@
...
+```
+
+##### environment-variables:
+```
+PX_HTTP_PROXY		[OPTIONAL] If running behind an HTTP proxy, set the PX_HTTP_PROXY variables to your HTTP proxy.
+PX_HTTPS_PROXY		[OPTIONAL] If running behind an HTTPS proxy, set the PX_HTTPS_PROXY variables to your HTTPS proxy.
+PX_ENABLE_CACHE_FLUSH	[OPTIONAL] Enable cache flush deamon. Set PX_ENABLE_CACHE_FLUSH=yes.
+```
+
+Setting environment variables can be done using the -e option. When generating the daemonset spec using -e query param (-eVAR=VAL,VAR=VAL,...).
```
Comment (harsh-px) [inline _includes/cmdargs.md]: This statement here is only required for k8s. It's sitting here in isolation with no k8s context. So will confuse end users.

File: scheduler/kubernetes/install.md [hunk 2]
```diff
@@ -124,14 +124,7 @@ $ kubectl label nodes minion5 "px/enabled=false" --overwrite
 Portworx is deployed as a [DaemonSet](https://kubernetes.io/docs/concepts/workloads/controllers/daemonset/). Therefore it automatically scales as you grow your Kubernetes cluster.  There are no additional requirements to install Portworx on the new nodes in your Kubernetes cluster.
 
 #### Installing behind the HTTP proxy
```
Comment (harsh-px) [inline scheduler/kubernetes/install.md]: Need to maintain the existing k8s spec generation example here. The redirect to the common page ` [Passing environment variables](/runc/options.html#environment-variables` is too generic and k8s users will get confused.

Comment (jrivera-px) [reply scheduler/kubernetes/install.md]: Ok after talking to Harsh it makes sense to move the curl example for install.portworx.com back to the kube install pages.    Its moved back to cmdargs.md.   The link for passing env arguments is still there with the example.

File: _includes/cmdargs.md [hunk 3]
```diff
@@ -1,35 +1,71 @@
...
+PX_HTTPS_PROXY		[OPTIONAL] If running behind an HTTPS proxy, set the PX_HTTPS_PROXY variables to your HTTPS proxy.
+PX_ENABLE_CACHE_FLUSH	[OPTIONAL] Enable cache flush deamon. Set PX_ENABLE_CACHE_FLUSH=yes.
+```
+
+Setting environment variables can be done using the -e option.  During PX Runc command line install (-e VAR=VAL).
+```
+$ sudo /opt/pwx/bin/px-runc install -c MY_CLUSTER_ID -e PX_ENABLE_CACHE_FLUSH=yes -k etcd://myetc.company.com:2379 -s /dev/xvdb -x kubernetes -v /var/lib/kubelet:/var/lib/kubelet:shared
+```
+
+Or by manually adding the arguments to the .yaml configuration file.
```
Comment (harsh-px) [inline _includes/cmdargs.md]: Remove the .yaml file from here. This yaml is k8s specific.

File: scheduler/kubernetes/install.md [hunk 4]
```diff
@@ -128,14 +128,16 @@ If Portworx has already been deployed in your cluster:
...
+  ```
+  $ curl -o px-spec.yaml \
+    "http://install.portworx.com?c=mycluster&k=etcd://etcd.fake.net:2379&e=PX_HTTP_PROXY=<http-proxy>,PX_HTTPS_PROXY=<https-proxy>"
+  ```
 
-```
-$ curl -o px-spec.yaml \
-  "http://install.portworx.com?c=mycluster&k=etcd://etcd.fake.net:2379&e=PX_HTTP_PROXY=<http-proxy>,PX_HTTPS_PROXY=<https-proxy>"
-```
+Alternative ways for providing environment variables can be found at [passing environment variables](/runc/options.html#environment-variables).
```
Comment (harsh-px) [inline scheduler/kubernetes/install.md]: For k8s, there is no alternative way. This is the only way. So the statement should be:

To view list of all Portworx environment variables, go to [passing environment variables](/runc/options.html#environment-variables).

**Pull request merged**